K2Bridge/Startup.cs

```diff
@@ -156,6 +156,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 // Special treatment for certains requests that are intentionally not marked with the [ApiController] attribute.
                 endpoints.MapControllerRoute("fieldcaps", "FieldCapability/Process/{indexName?}", defaults: new { controller = "FieldCapability", action = "Process" });
                 endpoints.MapControllerRoute("indexlist", "IndexList/Process/{indexName?}", defaults: new { controller = "IndexList", action = "Process" });
```

Comment:
Is this line still needed? (and the Process method itself)

Reply:
Removed.